src/plugins/autofill/autofill.js

```diff
@@ -162,7 +162,9 @@ class Autofill extends BasePlugin {
     if (startOfDragCoords && startOfDragCoords.row > -1 && startOfDragCoords.col > -1) {
       const selectionData = this.getSelectionData();
 
-      this.hot.runHooks('beforeAutofill', startOfDragCoords, endOfDragCoords, selectionData);
+      if (this.hot.runHooks('beforeAutofill', startOfDragCoords, endOfDragCoords, selectionData) === false) {
+        return false;
```

Comment:
`false` is unnecessary, sole `return` is enough to quit from `fillIn` function.

Reply:
`false` is returned in the first `if` statement so doing this would just keep it consistent.

Reply:
Maybe the cleaner and more consequence option is to save the result of this hook in const `const beforeAutofillHook = this.hot.runHooks('beforeAutofill', startOfDragCoords, endOfDragCoords, selectionData);` and the use it in `if` statement.

We do so in other places e.g. - https://github.com/handsontable/handsontable/blob/master/src/plugins/manualColumnMove/manualColumnMove.js#L192

As I said earlier and to be consistent `false` is unnecessary, `return` is enough ;)

Reply:
Okay, I will update the other `if` statement to just be a `return` at the same time so that it is consistent in the function too.

Reply:
P.S. The method's docblock says it returns a boolean.